.github/workflows/ci.yml

```diff
@@ -47,6 +41,7 @@ jobs:
               echo '_KERL_PREFIX_GIT=git' >> $GITHUB_ENV
               echo '_KERL_PREFIX_GIT_TARGET=https://github.com/erlang/otp.git' >> $GITHUB_ENV
               echo 'KERL_BUILD_DOCS=true' >> $GITHUB_ENV
+              _VERSION=git
```

Comment:
Without this there's be a mismatch on the generated scripts, since internally we use `git` for branch-based builds, not the branch itself.